src/js/modules/core/components/PageFooter.js

```diff
@@ -92,14 +102,39 @@ const PageFooter = ({ classes, topLevelSectionsWithDirectChildren }) => {
       );
     });
   };
+  const createLinksToDescriptions = () => {
```

Comment:
Rename `createLinksToDescriptions` to `createDescriptionLinks`. This has already been done with `createLinksToSections` in a separate module.